textbook/11/1/causality.ipynb

```diff
@@ -0,0 +1,235 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    dat = pd.DataFrame({'X':x,'Y':y, 'Z':z})
</pre><p> </p><p>…And this might be:</p><p> </p><pre class="ql-syntax" spellcheck="false">data = pd.DataFrame({'X': x, 'Y': y, 'Z': z})
</pre><p> </p><p>(<em>Etc</em>.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chicago-cdac/textbook-datascience-1/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>